### PR TITLE
Swedish.xml aligned with english.xml, fixed typos and corrected translations

### PR DESCRIPTION
**Pull request by ordtrogen**
I'm afraid this PR might be difficult to review.

I wanted to review the Swedish localization and to be able to compare it with english.xml, I first aligned swedish.xml with english.xml, i.e shuffled around the lines to put each swedish item on the same line in the file as in english.xml. Then used Winmerge to look at the files side-by-side.

Then fixed a couple of spelling errors, inconsistent terminology and some corrections.

### CODE REVIEW

**Comment by magol** [comment]
I strongly support this. I agree that it is hard to keep the Swedish translation in sync with the English. 

But it is, as you say, hard to review. Is it possible to do the sorting in one commit, and all fixes of spelling and inconsistencies in a separat commit (or even a separat PR).

**Comment by ordtrogen** [comment]
Hej Magnus och tack för allt jobb du lagt ner på översättningen.

TL;DR (see longer argument below)
The easiest way to revise at this point would probably be to use a program such as WinMerge and look at english.xml and the new swedish.xml side-by-side and see if you disagree with any of it:

image


Briefly what happened is that I saw a string in English in Arkiv-menyn and thought it was a new one that hadn't been translated yet and decided to do it. Then I discovered that the source and target files were not aligned, and did that instead, only to realise there were no untranslated strings...

At this point I should have saved a copy, but instead went on to do a revision of the translation as it stands, and we ended up in the present situation. 

I can see from the file history you've done a lot of work on this and let me state I was impressed with the translation when I worked through the file, and if you relied on connecting source segments with target segments using only the numerical identifier, that's pretty amazing. I know what it's like when you spend time on something and then someone shows up to "improve it". Annoying for sure. But there were no major issues. Here's an non-exhaustive list of the changes. Let me know what you think.

Both the concept "stora/små bokstäver" and "versaler/gemener" were used. I changed to only "versaler/gemener" and simplified somewhere to use "skiftläge"

"Radera" / "Ta bort" were both used -> "Ta bort" rakt igenom

Shortened some phrases, such as "den aktuella filen" -> "aktuell fil"

Mellanslag framför :  (removed) 
It looks like they were added at one point, I'm guessing because they are present in the source. But as you can see from (#4628), I posit they shouldn't be there, and then again it doesn't make sense to have a space before : in some strings and not in others.

"teckensnitt" / "typsnitt" -> enbart "teckensnitt"

Förlorad fokus -> förlorat fokus

Cancel in source has "Stäng" in target instead of "Avbryt", maybe there's a good reason for this, but I think I changed "Stäng" to "Avbryt" somewhere.

"...dess..." -> "...sina..." 

"Skapa &amp;bokmärke" -> "Skapa/ta bort &amp;bokmärke"  (since it says 'toggle' in source)

"urklipp" -> "Urklipp"

"Dokumentsvy" -> "Dokumentvy"

"...område..." -> "...intervall..."

"Upprepa" -> "Gör om"

"Angränsare [2-8]" -> "Avgränsare [2-8]

"...din Papperskorg..." -> "...Papperskorgen..."

siffra -> siffror (id 25028)

Matchade par [1-3] -> Matchat par [1-3]

**Comment by magol** [comment]
Even when you do your best, it is hard to do it perfekt. It is greate to have someone to review my work. I agree on all you sugested changes.

Jag har vid ett tidigare tillfälle försökt sortera strängarna, precis som du gjort nu, men tyvärr accepterades aldrig min PR. Jag hoppas verkligen att det går genom den här gången, allt skulle bli så mycket lättare då.

**Comment by magol** [comment]
I have now reviewed your changes. Your changes to the language are great. You've found some embarrassing spelling mistakes I've made, but also improved the language overall.

Your work on harmonizing the file with the English file will greatly simplify future work with the Swedish translation. However, I found that you missed to sort the strings in the `Scintillas `tag.

**Comment by ordtrogen** [comment]
I'm looking at the 'Scintillas' element right now and comparing swedish.xml and english .xml. The 'Item' elements are not in numerical order based on the 'id' attribute, that's true. But they are in the same order in both files. (6216, 6217, 6219, 6221, 6222, 6224, 6225, 6201, 6202, 6203, 6204, 6205, 6226 ...)

I was thinking maybe english.xml had been updated since my PR, but it says "4 months ago" when looking at in in GitHub, so I'm not sure what you mean.

**Comment by magol** [comment]
@ordtrogen Sorry, I did a mistake when when I did the compare in WinMerge. 
I can confirm that all you work is of great quality :-)

**Comment by donho** [comment]
Thank you @magol for the approvement!
